app/assets/javascripts/client/views/assessments/permissions/manage_permission.html

```diff
@@ -60,7 +60,7 @@
             <p class="info">
               <fieldset>
                 <label class="level_label">Current Permission Level: </label>
-                <select class="permission-level" name="level">
+                <select class="permission-level" name="level" ng-disabled="user.role === 'network_partner'">
```

Comment:
Nice catch, I initially didn't wanted to just catch the Network Partner requesting access piece(like I did in the PR), I also wanted to make it impossible for a network partner to become a participant, however, I thought it'd be too restrictive internally and decided instead to disable the option in the UI.

What I forgot was to restrict the permissions api to perform the change.

So instead limiting the list of role options for a network partner, I'd rather lock down the permissions API to prevent a network partner from becoming something else than a facilitator even if the user removes `disabled` from the **select HTML** element.

Thoughts?